lib/ASRouterTargeting.jsm

```diff
@@ -176,6 +179,12 @@ const TargetingGetters = {
   },
   get totalBookmarksCount() {
     return TotalBookmarksCountCache.getTotalBookmarksCount;
+  },
+  get firefoxVersion() {
+    return parseInt(AppConstants.MOZ_APP_VERSION, 10);
```

Comment:
how about `return parseInt(AppConstants.MOZ_APP_VERSION.match(/\d+/), 10);` to make this more robust